pgx/src/datum/sql_entity_graph/aggregate.rs

```diff
@@ -301,7 +302,7 @@ impl ToSql for PgAggregateEntity {
             "\n\
                 -- {file}:{line}\n\
                 -- {full_path}\n\
-                CREATE AGGREGATE {schema}{name} ({args}{maybe_order_by})\n\
+                CREATE OR REPLACE AGGREGATE {schema}{name} ({args}{maybe_order_by})\n\
```

Comment:
This seems unrelated?

Reply:
Ah indeed, didn't intend to add that change in this PR, apologies! I'll skim through and make sure other unrelated changes didn't also get added accidentally.

Reply:
I didn't see any! I was trying to figure out why that test was failing. :)

Reply:
`CREATE OR REPLACE AGGREGATE` is not valid in pg10 :( https://www.postgresql.org/docs/10/sql-createaggregate.html